src/nvim/eval.lua

```diff
@@ -8419,6 +8419,21 @@ M.funcs = {
     params = { { 'buf', 'integer|string' }, { 'text', 'string' } },
     signature = 'prompt_setprompt({buf}, {text})',
   },
+  prompt_getinput = {
```

Comment:
These functions are sorted alphabetically, which means this should come between `printf()` and `prompt_getprompt()`.